e2e/module-name-mapper-mock/src/storage/track/Track.js

```diff
@@ -0,0 +1,8 @@
+module.exports.Track = class Track {
+  constructor() {
+    /**/
```

Comment:
Isn't `someRandomFunction` supposed to be called here?

Reply:
No - we just check if it's a mock function or not, we don't invoke it

Reply:
Can possibly remove the constructor all together